TcUnit-Runner/TcUnitResultCollector.cs

```diff
@@ -228,7 +228,7 @@ public TcUnitTestResult ParseResults(IEnumerable<ErrorList.Error> errors, string
                                 if (expectedErrorLogEntryType != ErrorLogEntryType.TEST_NAME)
                                     log.Error("ERROR: While parsing TcUnit results, expected " + expectedErrorLogEntryType.ToString() + " but got " + ErrorLogEntryType.TEST_NAME.ToString());
                                 else
-                                    log.Error("ERROR: While parsing TcUnit results, got test case number " + currentTestCaseInTestSuiteNumber + " but expected amount is " + testSuiteNumberOfTests);
+                                    log.Error("ERROR: While parsing TcUnit results, got test case number " + currentTestCaseInTestSuiteNumber + " but expected number is " + testSuiteNumberOfTests);
```

Comment:
Hm... I'm not sure whether this will actually brake anything. TcUnit sends stuff over to the runner simply by the event-logger, and the format has to match. Have you tested to run the TcUnit-Runner with a set of tests?

Reply:
No I haven't. So far never used the runner. A good opportunity to test it out!

Reply:
No problems! I'll do the merge later tonight and just test that the Runner still gets results from TcUnit!